rdf-star-cg-spec.html

```diff
@@ -231,6 +231,30 @@ <h2>Concepts and Abstract Syntax</h2>
     </ul>
 
     <p>Again, this definition is an extension of the notion of <dfn data-cite="RDF11-CONCEPTS#dfn-rdf-dataset">RDF dataset</dfn>, hence it follows that any <a>RDF dataset</a> is also an <a>RDF* dataset</a>.</p>
+
+    <section class="informative" id="occurrences">
+      <h2>Triples and occurrences</h2>
+
+      <p>According to the definitions above, an <a>RDF* triple</a> is an abstract entity whose identity is entirely defined by its subject, predicate and object. Conversely, given three <a>RDF* terms</a> |s|, |p|, and |o|, there is exactly and only one <a>RDF* triple</a> with subject |s|, predicate |p| and object |o|. This unique triple (|s|, |p|, |o|) can be <a>embedded</a> as the subject or object of multiple other triples, but must be assumed to represent the <em>same thing</em> everywhere it occurs, including across different graphs, just like the same IRI |p| is assumed to represent the same thing everywhere it occurs.</p>
+
+      <p>In some situations, however, it might be necessary to distinguish the <em>occurrences</em> of a triple in different graphs. Consider the following sentence: "The triple `&lt;http://example.org/s&gt; &lt;http://example.org/p&gt; &lt;http://example.org/o&gt;` was added in (the graph represented by) file1.ttl by Alice, and the same triple was added in file2.ttl by Bob." Note that the words "same triple" in this sentence may be confusing, because although the triple (as an abstract entity) is the same, its respective occurrences are different things, each within a different file and with a different author (this is known, in philosophy and linguistics, as the <a href="https://en.wikipedia.org/wiki/Type%E2%80%93token_distinction">type-token distinction</a>). As the embedded triple represents a unique abstract entity, adequately conveying the meaning of the sentence above requires additional nodes for representing the distinct occurrences. One possible solution is illustrated in the following example (using the Turtle* concrete syntax described in <a href="#turtle-star">the next section</a>).</p>
```

Comment:
```suggestion
      <p>In some situations, however, it might be necessary to distinguish the <em>occurrences</em> of a triple in different graphs. Consider the following sentence: "The triple `&lt;http://example.org/s&gt; &lt;http://example.org/p&gt; &lt;http://example.org/o&gt;` in (the graph represented by) file1.ttl was added by Alice, and the same triple in file2.ttl was added by Bob." Note that the words "same triple" in this sentence may be confusing, because although the triple (as an abstract entity) is the same, its respective occurrences are different things, each within a different file and with a different author (this is known, in philosophy and linguistics, as the <a href="https://en.wikipedia.org/wiki/Type%E2%80%93token_distinction">type-token distinction</a>). As the embedded triple represents a unique abstract entity, adequately conveying the meaning of the sentence above requires additional nodes for representing the distinct occurrences. One possible solution is illustrated in the following example (using the Turtle* concrete syntax described in <a href="#turtle-star">the next section</a>).</p>
```

Reply:
I think the given example sentence is somewhat misleading. By the way it is phrased, it is not entirely clear that this sentence is meant to about the occurrence of the triple rather than about the triple itself. In other words, when I read the sentence, I think: "Ah! Alice had this triple and she added it to the graph/file; additionally, Bob also had this triple and he added it to the other graph/file." So, based on this reading, it seems to me that the sentence is more about the triple itself rather than about its different occurrences. Now, my suggested change of the example sentence should fix this issue because it talks much more explicitly about the occurrences.